zulipterminal/helper.py

```diff
@@ -49,6 +50,12 @@
 StreamAccessType = Literal["public", "private", "web-public"]
 
 
+class SearchStatus(Enum):
+    DEFAULT = 0
+    FILTERED = 1
+    EMPTY = 2
+
```

Comment:
I didn't add any because none of the other classes in helper.py have docstrings.
What aspect do you think this could be more clear about? We could rename it too.

Reply:
Seems fair. I was just thinking it might be confusing for people looking at the code later, so a docstring would help make it clearer, but now that you mention it, it might not be necessary.

Reply:
Seems fair. I was just thinking it might be confusing for people looking at the code later, so a docstring would help clarify it. However, now that you mention it, it might not be necessary.

Reply:
Good naming > docstrings/comments, but it could be helpful if the naming can't be easily improved - and it's good not to block on getting perfect names ;)